packages/frontend/src/components/common/ShareSpaceModal/index.tsx

```diff
@@ -91,65 +85,95 @@ const ShareSpaceModal: FC<ShareSpaceProps> = ({ space, projectUuid }) => {
             >
                 <>
                     <Stack p="md" pt={0}>
+                        {isNestedSpace && (
+                            <Alert
+                                color="blue"
+                                icon={<IconAlertCircle size="1rem" />}
+                            >
+                                <Text color="blue.9">
+                                    <Text span weight={600}>
+                                        {space.name}
+                                    </Text>{' '}
+                                    inherits permissions from it's root space:{' '}
+                                    <Text span weight={600}>
+                                        <Link
+                                            onClick={() => {
+                                                setIsOpen(false);
+                                            }}
+                                            to={`/projects/${projectUuid}/spaces/${rootSpaceBreadcrumb?.uuid}`}
```

Comment:
is there a case when `rootSpaceBreadcrumb?.uuid` this is not defined? because it's gonna output undefined